### PR TITLE
Issue 44797: Update data alignment with custom and standard participant views

### PR DESCRIPTION
**Pull request by RosalineP**
#### Rationale
When you customize the participant view in a study, our default code is showing a particularly wide first column. By default, the first column should be within a reasonable width for readability.

#### Related Pull Requests
* N/A

#### Changes
* Update styling

### CODE REVIEW

**Comment by RosalineP** [comment]
Cory is the lucky individual chosen for review, as he touched the adjacent styling most recently!

**Review by labkey-mohara — APPROVED** [review]
Looks great

File: study/src/org/labkey/study/view/participantAll.jsp [hunk 1]
```diff
@@ -239,6 +239,11 @@
     .labkey-ptid-remove, .labkey-ptid-add {
         cursor: pointer;
     }
+
+    .labkey-data-region tr td:first-child
+    {
+        width: 300px;
```
Comment (cnathe) [inline study/src/org/labkey/study/view/participantAll.jsp:245]: This code change looks good and makes sense. I pulled down the fb to test it, and it worked good for the default case. However, if I changed the dataType from ALL to DEMOGRAPHIC, this column width was no longer getting applied. I'm not sure why. Thoughts?
Screen Shot 2022-07-04 at 9 46 03 AM

**Review by cnathe — APPROVED** [review]

File: study/src/org/labkey/study/view/participantAll.jsp (hunk 1 above)
Comment (RosalineP) [reply study/src/org/labkey/study/view/participantAll.jsp:245]: Ah, great catch! The page was switching views between the participantAll.jsp and customizeParticipantView.jsp, the latter of which the style was not in. Update made, and the fix is now working across cases.

Comment (RosalineP) [reply study/src/org/labkey/study/view/participantAll.jsp:245]: Great catch! The customized participant view was stomping over the changes. Newest commit works for all cases (except the case where a client has a pre-existing customized participant view, in which case that custom view is preserved.

**Pull request merged**